### PR TITLE
Add job_time

### PR DESCRIPTION
**Pull request by KetpuntoG**

### CODE REVIEW

**Comment by antalszava** [comment]
[sc-11508]

File: pennylane_qiskit/qiskit_device.py [hunk 1]
```diff
@@ -397,6 +396,9 @@ def batch_execute(self, circuits):
         self._current_job = self.backend.run(compiled_circuits, shots=self.shots, **self.run_args)
         result = self._current_job.result()
 
+        if self.tracker.active:
```
Comment (rmoyard) [inline pennylane_qiskit/qiskit_device.py:399]: Could we move this to line 416?

```
if self.tracker.active:
            self._track_run()
            self.tracker.update(batches=1, batch_len=len(circuits))
            self.tracker.record()
```

**Review by rmoyard — COMMENTED** [review]
@KetpuntoG Thanks for the PR, it is working as expected but I don't like the empty function for `qiskit_device`. Could we delete `_track_run` in the `qiskit_device` and redefine `batch_execute` in the `IBMQDevice`. Something like

```
def batch_execute(self, circuits):
         super().(circuits)
         if self.tracker.active:
                  self._track_run()
```

should be working, let me know if it is the case !

**Comment by rmoyard** [comment]
@KetpuntoG Great changes, can you just add it to the changelog 💯

**Comment by KetpuntoG** [comment]
Now I'll add to the chagelog :) I find it strange that I pass the tests in python 3.7 and 3.9 but not in 3.8, do you know why it could be @rmoyard ?

**Comment by KetpuntoG** [comment]
#109

**Review by rmoyard — APPROVED** [review]
Thanks @KetpuntoG it looks good to me 👍

**Pull request merged**